### ISSUE TITLE
Need to update S1 AFT map and cut again?

### ISSUE THREAD

**Issue by pdeperio** [XENON1T/pax#609]
@darrylmasson Does @l-althueser's S1 pattern map update (https://github.com/XENON1T/pax/pull/603, [note](https://xecluster.lngs.infn.it/dokuwiki/doku.php?id=xenon:xenon1t:sim:notes:althueser:xe1t_msc_mc_perpmt#Comparison-of-mc-v0.2.1/v0.3.0-with-new-binning)) answer one of the issues in [your note](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:darryl:s1_aft_update) (Figure 5b)?  If so, do you need to [update the S1 AFT map](https://github.com/XENON1T/pax/pull/598) again?  Then can you please update your note with this and the following:
- Did you figure out what's going on at s1~7 pe?
- Understand the acceptance loss at high energy and fix/tune the cut accordingly?  Necessary for [ongoing high energy studies (Fig. 4.2)](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:analysis:sciencerun1:g1_g2_systematics#effect_of_different_cut_sets).

**Comment by darrylmasson** [comment]
A couple of things:

- After #603 was merged, I ran another half million waveforms through fax and saw how things differ. The kink seen in Fig 5b in my note that starts at -8cm is gone so the overall behavior is now much more similar to Fig 5a. However, the median lines still don't match. Fax still tends to have a more extreme trend (lower area_fraction_top at the cathode than data, and higher at the gate), and this hurts the acceptance.
- The map used in the S1AreaFractionTopProbability plugin shouldn't need to be changed - it's data-driven so shouldn't be subject to these issues in the simulation.
- The notch at 7 PE is from events with area_fraction_top = 0 in regions where that doesn't happen much in data. These are mainly events in the center of the population in Fig 5b that are left of the red line - these events have a systematically lower area_fraction_top than as seen in data. I'm not exactly sure on the very root cause, but after #603 the depth of the notch has decreased slightly (1% improvement or so). 
- I am not confident in the performance of this cut at high energies. Once saturation becomes significant, if we can't desaturate to better than 1% error, it just won't work well. At high energies the band in z-aft space is very narrow, and deviations from the trend are heavily disfavored. When I get around to it I intend to relist the cut in lax as a "low-energy" cut, or at least put a caveat in the docstring that the acceptance does decrease significantly above 1000 PE.

Overall #603 doesn't change much in the FV. I have a couple of ideas on how to use data to check the acceptance out to high energies (rather than fax), but haven't had the chance to work on them.
